scrapers/scrapeDepartmentCourses.js

```diff
@@ -226,16 +233,24 @@ const parseIntervalLine = line => {
   const isWebBased = specialEnrollment.includes("WEB BASE")
   const isComIntensive = specialEnrollment.includes("CI-WRITTEN&SPOK")
 
-  const instructorName = line.slice(117, line.length - 1)
+  const instructorName = line.slice(117, line.length)
+
+  // Fix fields
+
+  if (Number.isNaN(enrollmentAvailable)) enrollmentAvailable = 0
+
+  console.log(enrollmentFull)
+
+  // Return final object
 
   return {
     enrollment: {
-      available: enrollmentFull ? 0 : enrollmentAvailable,
+      available: enrollmentAvailable,
```

Comment:
Nevermind, I see you addressed that in the lines above. But I wonder if the ternary was better here?